src/libs/ReportUtils.ts

```diff
@@ -1221,6 +1221,33 @@ function isOneOnOneChat(report: OnyxEntry<Report>): boolean {
     );
 }
 
+/**
+ * Checks if the current user is a payer of the request
+ */
+
+function isPayer(
+    policy: OnyxEntry<Policy> | EmptyObject,
+    session: OnyxEntry<Session>,
+    iouReport: OnyxEntry<Report>,
+    // eslint-disable-next-line @typescript-eslint/no-shadow
+    isPaidGroupPolicy: boolean,
```

Comment:
I think we can use the function `PolicyUtils.isPaidGroupPolicy(policy)` inside of `isPayer` so it won't be necessary to pass `isPaidGroupPolicy` as prop and diable the eslint rule